src/CodeGen/ODataT4CodeGenerator.cs

```diff
@@ -345,6 +349,12 @@ public enum LanguageOption
     VB = 1,
 }
 
+public Dictionary<string, string> CustomHttpHeaders

```

Comment:
Maybe use "IDictionary<string, string> is better? and mark it as virtual?

Reply:
I have replaced with `virtual IDictionary<string, string>` so as to allow *mocking*